internal/service/advancedclustertpf/data_source.go

```diff
@@ -14,6 +14,12 @@ import (
 var _ datasource.DataSource = &ds{}
 var _ datasource.DataSourceWithConfigure = &ds{}
 
+const (
+	errorReadDatasource                 = "Error reading  advanced cluster datasource"
+	errorReadDatasourceAsymmetric       = "Error reading  advanced cluster datasource"
```

Comment:
these errors seem the same. Why do you need both?

Reply:
Thanks. Changed in 13638b93527c2a25df1d139b4e6ab669252985ae